make/scripts/fixpath.sh

```diff
@@ -149,7 +149,8 @@ function import_path() {
     winpath="$($PATHTOOL -w "$path" 2>/dev/null)"
     # If it fails, try again with an added .exe (needed on WSL)
     if [[ $? -ne 0 ]]; then
-      winpath="$($PATHTOOL -w "$path.exe" 2>/dev/null)"
+      path="$path.exe"
```

Comment:
This seems to conflict with the logic starting at line #173 below.

Reply:
I pushed to new commit to save original `$path` and restore it later if the path not found.
Could you review again?

Reply:
Sorry but I'm really not following exactly what the problem is and thus the solution. How does setting path affect the running of PATHTOOL? Does the problem indicate we should be using winpath rather than path somewhere?

Reply:
In this case `$PATHTOOL` points `wslpath`, and it tries to convert `$JAVA_HOME/bin/java` to Windows path. However `java` on Windows is `java.exe`, so we need to convert it.

Currently `fixpath.sh` adds `.exe` if `wslpath` fails, then attempt to do it again, but `$path` still lost extension ( `.exe`). So `configure` will fail on WSL because `java.exe` cannot be find.